### DISCUSSION TITLE
Graduated and need to switch from Student to Educator Account

### DISCUSSION THREAD

**Discussion by rnyang** [community/community#88608] [GitHub Education]
### Hi everyone,

Hi folks. I've recently graduated and need to switch from a Student to an Educator account. However, when I try to apply for benefits as a teacher (https://education.github.com/discount_requests/application), I can select my new teacher email, but cannot adjust the field under "What is the name of your school?".

Any suggestions? Thanks!

**Comment by Shahnawazk7944** [comment]
You can try as an independent educator.
